ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -26,7 +30,8 @@ export default function GasTiming({ maxPriorityFeePerGas }) {
   };
 
   // Don't show anything if we don't have enough information
-  if (isGasEstimatesLoading) {
+  // or if the network doesn't support 1559
```

Comment:
Just realized that there is a better / more robust check that we can do here. Instead of checking whether the network supports EIP-1559, we should check if `gasEstimateType === GAS_ESTIMATE_TYPES.FEE_MARKET`

`gasEstimateType` is available from `useGasFeeEstimates()`, which is already called above, and `GAS_ESTIMATE_TYPES` will have to be imported from the shared constants.

The reason we should do this check, instead of just the network check, is that if for some reason the request for fee market type estimates failled, and the gas controller fell back to just requesting the gas price, then there will be no `low, medium and high` available on `gasFeeEstimates` and the below code will break